### PR TITLE
ESCBackground node is no more necessary

### PR DESCRIPTION
**Pull request by StraToN**
This PR removes all ESCBackground nodes from rooms and Escoria will now add it on room loading and set its size using the first value in ESCRoom's camera_limits.

This means that the FIRST element in camera_limits must be set to the WHOLE scene size.

### CODE REVIEW

File: addons/escoria-core/plugin.gd [hunk 1]
```diff
@@ -301,6 +301,14 @@ func set_escoria_debug_settings():
 			"hint": PROPERTY_HINT_DIR
 		}
 	)
+	
+	register_setting(
```
Comment (BHSDuncan) [inline addons/escoria-core/plugin.gd:305]: Does this need to be here? Wasn't it already merged into `dev`?

File: project.godot [hunk 2]
```diff
@@ -204,6 +204,11 @@ _global_script_classes=[ {
 "language": "GDScript",
 "path": "res://addons/escoria-core/game/scenes/dialogs/esc_dialog_player.gd"
 }, {
+"base": "ESCDialogManager",
```
Comment (BHSDuncan) [inline project.godot:207]: Like the last PR...is this ok to leave in? If so, no worries.

File: game/translations/main_menu.de.translation
Comment (BHSDuncan) [inline game/translations/main_menu.de.translation:1]: Should we remove these translation file edits?

**Review by BHSDuncan — CHANGES_REQUESTED** [review]

File: addons/escoria-core/game/core-scripts/esc_room.gd [hunk 3]
```diff
@@ -76,9 +76,12 @@ func _ready():
 	for child in get_children():
 		if child is ESCBackground:
 			found_escbackground = true
+			move_child(child, 0)
 	if not found_escbackground:
 		var esc_bg = ESCBackground.new()
-		esc_bg.name = "background"
+		esc_bg.name = "escbackground"
```
[the remark this answers is not in the document]
Comment (StraToN) [reply addons/escoria-core/game/core-scripts/esc_room.gd:82]: Right. Done

File: project.godot (hunk 2 above)
Comment (StraToN) [reply project.godot:207]: Indeed, removing.

File: game/translations/main_menu.de.translation
Comment (StraToN) [reply game/translations/main_menu.de.translation:1]: Indeed. Removing as well.

File: addons/escoria-core/plugin.gd (hunk 1 above)
Comment (StraToN) [reply addons/escoria-core/plugin.gd]: Right this was duplicated for some reason.

**Review by BHSDuncan — APPROVED** [review]

**Pull request merged**